For #5185: Refactor about page.

mcarare opened the pull request:
Comparison to old screen:

<img src = https://user-images.githubusercontent.com/48995920/131317641-be5fb1c0-4b64-4d55-91d5-bd747585e844.png width = 49%> <img src = https://user-images.githubusercontent.com/48995920/131134692-e3ba3033-501d-443f-8d58-878c6133c4a6.jpg width = 49%>

brampitoyo commented:
@mcarare Good call, and good-looking design, too!

A couple of suggestions:

1. Is the logo the same size as the one found on the New Tab Page? Perhaps it’s better if the size is consistent everywhere.
2. Let’s make sure that the text style and colour matches one we use elsewhere in the app, and matches the Fenix about page. What’s the line-height value of the old design, by the way?
3. The second line of each bullet list item should be indented, so it sits together with the first line.

mcarare commented:
@brampitoyo I have updated the screenshot for the new version:

1. I have removed the small padding from the logo so it shows in the original size (same as in the splash screen).
2. I have added the line height specified in the HTML.
3. There isn't much I can do about that list, it is a string formatted for being displayed in a HTML file. I removed the tabulation before the bullet symbol and now the text wraps, but without the space before. IMO it looks better. For future iterations, when I can add new strings I can separate that part and format it better, but for now, it is riskier to split a localized string into so many parts.

cadeyrn commented:
> IMO it looks better.

IMO it was previously more pleasant to read. Now the list is a bit cramped due to the reduced spacing and line height…

mcarare commented:
@brampitoyo I can also replace &#x2022; with a bigger dot,  &#x25CF; if neccessary.

pocmo commented on app/src/main/res/layout/fragment_about.xml:
```diff
@@ -0,0 +1,36 @@
+            android:layout_height="match_parent" />
+
+    </LinearLayout>
+</LinearLayout>
```
nit: Would be nice to have no layout here at all. But makes sense with the toolbar for now, I guess. Maybe we can file a follow-up. :)

mcarare replied:
I guess eventually we should consider having a screen composable that automatically includes a toolbar and we can migrate more screens to compose.

pocmo commented on app/src/main/java/org/mozilla/focus/fragment/AboutFragment.kt:
```diff
@@ -0,0 +1,182 @@
+                .replace("<p>", "\n")
+                .replace("</p>", "")
+                .replaceAfter("<br/>", "")
+                .replace("<br/>", "")
```
This feels a bit hacky, given that we can't really control the number of line breaks in the translations. Maybe we should consider splitting the strings. Especially if it also allows us to improve the layout. And it will confuse translators less...

Maybe we can ask @Delphine if that would be okay, I'll approve here anyways since we could try this for now (but we should look at some of the translations to see if that splitting approach works)

mcarare replied:
Yes, this was my initial proposal, but it has been postponed until after MVP when we would eventually even change the text on this screen.

gabrielluong commented on app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt:
```diff
@@ -672,6 +672,16 @@ class UrlInputFragment :
     }
 
     private fun openUrl(url: String, searchTerms: String?) {
+
```
nit: no new line here

brampitoyo reviewed (approved):
Looks good to me! But I also agree with @cadeyrn that we should fix the alignment of the bullet points, when we’re able to, in the future.
